### DISCUSSION TITLE
FileNotFoundError - version.txt

### DISCUSSION THREAD

**Discussion by amitli1** [s3prl/s3prl#516] [General]
I'm trying to load wavlm_large model with s2prl (torch.hub.load) and getting error:

```
import torch
import os

pp = os.environ.get("S3PRL_MAIN_PATH", : '"home/usr/.cache/torch/hub/s3prl_s3prl_main"

torch.hub.load(repo_or_dir=pp, model="wavlm_large", source="local", trust_repo=True, force_reload=False,verbose=True,skip_validation=True)

```
Error:
`FileNotFoundError : [Error 2] No such file or directory: '/home/usr/.cache/torch/hub/s3prl_s3prl_main/s3prl/version.txt'`

What am I missing ?

**Comment by leo19941227** [comment]
Hi, could you try to use our latest package-based usage?

See: https://s3prl.github.io/s3prl/tutorial/upstream_collection.html
How to install: https://s3prl.github.io/s3prl/tutorial/installation.html

This usage is more reliable and we do not ensure the torch.hub usage always work.
Thanks!
